batavia/builtins/hash.js

```diff
@@ -1,8 +1,8 @@
 var exceptions = require('../core').exceptions
 var type_name = require('../core').type_name
-var types = require('../types')
 
 function hash(args, kwargs) {
+    var types = require('../types')
```

Comment:
@dimaqq I moved `var types = require('../types')` because I was otherwise unable to access `types.isinstance` inside of the `hash` function